pom.xml

```diff
@@ -118,8 +154,17 @@
                 <plugin>
                     <groupId>org.jenkins-ci.tools</groupId>
                     <artifactId>maven-hpi-plugin</artifactId>
+                    <extensions>true</extensions>
                     <configuration>
-                        <pluginFirstClassLoader>true</pluginFirstClassLoader>
+                        <pluginFirstClassLoader>false</pluginFirstClassLoader>
+                        <compatibleSinceVersion>2.9</compatibleSinceVersion>
+                        <dependentWarExcludes>org.apache.jclouds*</dependentWarExcludes>
+                        <archive>
+                            <manifestEntries>
+                                <Mask-Classes>com.google.common</Mask-Classes>
```

Comment:
> Leaving this value as true, the class jenkins.plugins.openstack.compute.OpenStack wont get loaded by the CPS classLoader.

There is a history for years futile attempts fiddling with all these options and none of them ever worked.